### PR TITLE
#208 split PR 2, rename fsys to fieldSystem

### PR DESCRIPTION
**Pull request by red031000**
split from #208 renames fsys to fieldSystem, Fsys to FieldSystem, and FieldSys to FieldSystem

### CODE REVIEW

**Comment by luckytyphlosion** [comment]
Do we actually want this? I thought the purpose of `fsys` was that the field is used so commonly that a short term is preferable.

**Comment by red031000** [comment]
short term is not preferable, especially if it's in function names

**Comment by luckytyphlosion** [comment]
That being said, I was under the impression that `fsys` was only used to refer to the variable, and FieldSystem was used everywhere else, which is my personal opinion for the naming.

**Comment by luckytyphlosion** [comment]
I agree that in function names only one term should be used. So the main questions are:
- In variables/struct fields/parameters: `fsys`, `fieldSys`, or `fieldSystem`?
- In function names (and other stuff): `Fsys`, `FieldSys`, or `FieldSystem`?

However, something like this isn't as broad as the other style issues, so we can choose whatever now and revisit it later. So basically it's up to you and @adrienntindall to decide.

**Comment by adrienntindall** [comment]
> I agree that in function names only one term should be used. So the main questions are:
> 
> * In variables/struct fields/parameters: `fsys`, `fieldSys`, or `fieldSystem`?
> * In function names (and other stuff): `Fsys`, `FieldSys`, or `FieldSystem`?
> 
> However, something like this isn't as broad as the other style issues, so we can choose whatever now and revisit it later. So basically it's up to you and @adrienntindall to decide.

Function names need to be FieldSystem but I prefer fsys as the local/struct variable names, however, I don't feel strongly about this

**Comment by red031000** [comment]
where possible I tend to prefer explicit names, usually the var name being the struct type in camelCase

File: include/field_use_item.h [hunk 1]
```diff
@@ -31,10 +31,10 @@ struct AlphItemUseData {
     u16 var_8003;
 };
 
-typedef void *(*FieldApplicationWorkCtor)(FieldSystem *fsys);
+typedef void *(*FieldApplicationWorkCtor)(FieldSystem *fieldSystem);
 
 struct ItemFieldUseData {
-    FieldSystem *fsys;             // 00
+    FieldSystem *fieldSystem;             // 00
```
Comment (adrienntindall) [inline include/field_use_item.h:37]: spacing of comment

File: include/field_follow_poke.h [hunk 2]
```diff
@@ -20,21 +20,21 @@ u8 sub_02069EC0(LocalMapObject *mapObject);
 u8 sub_02069ED4(LocalMapObject *mapObject);
 void FollowPokeMapObjectSetParams(LocalMapObject *mapObject, int species, u8 form, BOOL shiny);
 void sub_02069F0C(LocalMapObject *mapObject, int species, u8 form, BOOL shiny, int a4);
-void FollowPokeFsysParamSet(FieldSystem *fsys, int species, u8 form, BOOL shiny, u8 gender);
+void FollowPokeFieldSystemParamSet(FieldSystem *fieldSystem, int species, u8 form, BOOL shiny, u8 gender);
```
Comment (adrienntindall) [inline include/field_follow_poke.h:23]: FieldSystem_SetFollowPokeParam?

**Review by adrienntindall — CHANGES_REQUESTED** [review]

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**